### PR TITLE
Feature/osint 273

### PR DESCRIPTION
**Pull request by ahrufox**
# [OSINT-273] Maigret
## 🔑 Key Change 
- Node related with Maigret

## 🖼️ Screenshots (if necessary)
image

## 🙏 To Reviewers
🧑‍🤝‍🧑 @Teammate
- @D7MeKz thx

[OSINT-273]: https://bob-transfer.atlassian.net/browse/OSINT-273?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: server/osint/tools/lib/tool_maigret.py [hunk 1]
```diff
@@ -56,10 +58,21 @@ def report_maigret(run):  # status is COMPLETED
         "state": run.status,
         "results": result_list
     }
+
+    # node
+    s_user = SurfaceUser.get_node({'username': run.input_value, 'case_id': case_id})
+    if not s_user:
+        s_user = SurfaceUser.created_node({'username': run.input_value, 'case_id': case_id})
```
Comment (D7MeKz) [inline server/osint/tools/lib/tool_maigret.py:65]: Is it **create_node()** not created_node()?

**Review by D7MeKz — CHANGES_REQUESTED** [review]
Check it plz.

File: server/osint/tools/lib/tool_maigret.py (hunk 1 above)
Comment (ahrufox) [reply server/osint/tools/lib/tool_maigret.py]: oh shoot. but it worked... how...?

**Review by D7MeKz — APPROVED** [review]
:+1

**Pull request merged**